public/background.js

```diff
@@ -3,7 +3,7 @@ import { getData, storageSync } from './storage.js';
 import { increaseCommands, decreaseCommands } from './commands.js';
 
 // onInstalled
-chrome.runtime.onInstalled.addListener(function(details) {
+chrome.runtime.onInstalled.addListener(async function(details) {
```

Comment:
making this async allows for the use of await on line 22. Not needed, but a nicety.